packages/admin-ui/ui/src/utils/get-admin-path.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * returns the full admin path using the window object
+ * @returns {string} - admin path
+ */
+export function getFullAdminPath() {
+  if (window) {
+    const origin = window.location.origin // returns base url with trailing slash
+
+    const adminPath = process.env.ADMIN_PATH ? `${process.env.ADMIN_PATH}/` : ""
+
+    let fullPath = `${origin}${adminPath}`
+
+    // safeguard against double slashes, which happens in case path is set to `/`
+    if (fullPath.endsWith("//")) {
```

Comment:
Suggestion(non blocking): fullPath.replaceAll(´//´, ´/´)

To also prevent unwanted double slash anywhere

Reply:
I don't think we necessarily want to replace all. We just need to make sure, that whatever is appended to the full admin path is appended correctly i.e. with no double slashes.